index.html

```diff
@@ -359,31 +301,13 @@ <h2><code>PointerEvent</code> Interface</h2>
                         <dd>
                             <p>Indicates if the pointer represents the <a>primary pointer</a> of this pointer type.</p>
                         </dd>
-                     <dt><dfn>getCoalescedEvents</dfn></dt>
+                     <dt><dfn>getCoalescedEvents()</dfn> method</dt>
                         <dd>
-                            <p><a>coalesced event list</a>'s getter, when invoked, must run these steps:
-                            <ol>
-                                <li>If the <a>coalesced events targets dirty</a> is true:
-                                for each event in the <a>coalesced event list</a>, set the event's
-                                <a href="https://dom.spec.whatwg.org/#event-target"><code>target</code></a>
-                                to this <code>PointerEvent</code>'s target.</li>
-                                <li>Set the <a>coalesced events targets dirty</a> to false.</li>
-                                <li>Return the <a>coalesced event list</a>.</li>
-                            </ol>
-                            </p>
+                            <p>Getter method for <a>coalesced events</a>.</p>
```

Comment:
Is Getter method some explicitly defined thingie? at least it isn't a getter in JS sense. Should this be "A method to access coalesced events." or some such

Reply:
good point, i prefer that (was only going by the previous language that was there). makes it much more understandable.

Reply:
changed to "A method that returns the list of ..." in both cases.